python_modules/libraries/dagster-aws/dagster_aws_tests/pipes_tests/fake_ecs.py

```diff
@@ -130,12 +131,21 @@ def run_task(self, **kwargs):
 
         return response
 
-    def describe_tasks(self, cluster: str, tasks: List[str]):
+    def describe_tasks(self, cluster: str, tasks: List[str], WaiterConfig: Optional[dict] = None):
         assert len(tasks) == 1, "Only 1 task is supported in tests"
 
         simulated_task = cast(SimulatedTaskRun, self._task_runs[tasks[0]])
 
-        response = self.ecs_client.describe_tasks(cluster=cluster, tasks=tasks)
+        args = {
+            "cluster": cluster,
+            "tasks": tasks,
+        }
+
+        # The stub library doesn't support WaiterConfig as an argument so don't pass it to the stub
```

Comment:
I don't think `WaiterConfig` should be set here? not sure if this comment is relevant since it's an argument to `waiter.wait`, not `ecs_client.describe_tasks`

Reply:
Yep the `describe_tasks()` signature is very similar to the `wait()` signature so I just got them mixed up, removing.